rl4co/utils/meta_trainer.py

```diff
@@ -0,0 +1,111 @@
+import lightning.pytorch as pl
+import torch
+import math
+import copy
+from torch.optim import Adam
+
+from lightning import Callback
+from rl4co import utils
+import random
+log = utils.get_pylogger(__name__)
+
+
+class ReptileCallback(Callback):
+
+    # Meta training framework for addressing the generalization issue
+    # Based on Zhou et al. (2023): https://arxiv.org/abs/2305.19587
+    def __init__(self,
+                 num_tasks,
+                 alpha,
+                 alpha_decay,
+                 min_size,
+                 max_size,
+                 sch_bar = 0.9,
+                 data_type = "size",
+                 print_log=True):
+        super().__init__()
```

Comment:
[Documentation] It's recommended to have a doc with parameters, possibly including the data type, constraints, hints, etc. Better for us "non-experts" to understand 😆

```python
class ReptileCallback(Callback):
    """Meta training framework for addressing the generalization issue
    Based on Zhou et al. (2023): https://arxiv.org/abs/2305.19587

    Args:
        - num_tasks: number of task types, i.e. `B` in the original paper
        - alpha: ...
        - ...
    """

    def __init__(
        self,
        num_tasks: int,
        alpha: float,
        alpha_decay: float,
        min_size: int,
        max_size: int,
        sch_bar: float = 0.9,
        data_type: str = "size",
        print_log: bool = True,
    ):
        super().__init__()
```

Reply:
Hi Chuanbo. I have added the documentation as you recommended, along with the generation code for some distributions defined in the generalization-related works. Now the meta learning framework is supported for cross-distribution generalization.